### DISCUSSION TITLE
Regenerate API token

### DISCUSSION THREAD

**Discussion by kaleidawave** [rust-lang/crates.io#8414] [Feature Requests]
Currently once a token expires it sits on the dashboard

![image](https://github.com/rust-lang/crates.io/assets/26967284/19a9c58c-bf0f-44e2-8541-11d3a49afd5d)

Would be nice to have a "regenerate" button (in the same position the "revoke" button for valid tokens) so we don't have to create a new token, copy the name and scopes and crate patterns once a token expires

**Comment by Turbo87** [comment]
see https://github.com/rust-lang/crates.io/issues/8154 :)
